inference-engine/thirdparty/CMakeLists.txt

```diff
@@ -82,6 +82,19 @@ if (NOT USE_SYSTEM_PUGIXML)
 endif()
 
 if(ENABLE_MKL_DNN)
-    set(SDL_cmake_included ON)
-    include(mkldnn.cmake)
+    set(DNNL_ENABLE_CONCURRENT_EXEC ON CACHE BOOL "" FORCE)
+    set(DNNL_ENABLE_PRIMITIVE_CACHE OFF CACHE BOOL "" FORCE) ## TODO: try it later
+    set(DNNL_ENABLE_MAX_CPU_ISA OFF CACHE BOOL "" FORCE)     ## TODO: try it later
+    set(DNNL_LIBRARY_TYPE STATIC CACHE BOOL "" FORCE)
+    set(DNNL_BUILD_EXAMPLES OFF CACHE BOOL "" FORCE)
+    set(DNNL_BUILD_TESTS OFF CACHE BOOL "" FORCE)
+    set(DNNL_CPU_RUNTIME "TBB" CACHE BOOL "" FORCE)
```

Comment:
How such an explicit opt is compatible with the OMP codepath in the IE/CPU plugin?

Reply:
Fixed by https://github.com/openvinotoolkit/openvino/pull/3725/commits/4a6a84bc4397c91a919c7d442634495de9a17149 and https://github.com/openvinotoolkit/oneDNN/pull/22/commits/a8b86fa718b7f01df5ea8ababce7ff65ecbcdf38. 
FYI: OMP build is broken on master at the moment. Fix: https://github.com/openvinotoolkit/openvino/pull/3801